test/unit/services/oooRequest.test.ts

```
@@ -0,0 +1,175 @@
+                until: Date.now() + convertDaysToMilliseconds(3),
+                type: REQUEST_TYPE.OOO,
+                reason: testReason,
+            });
+        });
+
+        afterEach(async ()=> {
+            await cleanDb();
+            sinon.restore();
+        });
+
+        it("should return USER_STATUS_NOT_FOUND if user status not found", async () => {
```

ESLint

[error] Replace `········` with `····`

(prettier/prettier)